src/components/DoctorsPage/DoctorsPage.tsx

```diff
@@ -17,26 +17,32 @@ const DoctorsPage: React.FC = () => {
     dispatch(fetchDoctors());
   }, []);
 
-  const [choiseDirection, setChoiseDirection] = useState<string>('All Doctors');
+  const conditionRendering = () => {
+    if (doctors.loading) return <Loader />;
+    else if (doctors.error) return <Error errorMessage={doctors.error} />;
+    else {
+      return (
+        <DoctorsWrapper>
+          {choiseDirection.length === 1 && choiseDirection[0] === 'All Doctors'
```

Comment:
could you pls explain here the meaning of this condition?

Reply:
![image](https://user-images.githubusercontent.com/56519328/141755838-3bc5f29c-7b75-4971-b8c5-9aff7ae5dce5.png)
 i fixed